apps/backend/src/utils/nginx_parser.py

```
@@ -0,0 +1,553 @@
+"""
+Nginx Configuration Parser
+
+Utilities for parsing and analyzing nginx configuration files
+from SWAG reverse proxy setups.
+"""
+
+import re
+import logging
+from typing import Dict, List, Optional, Any, Tuple
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)